frontend/src/components/UserDetails.vue

```diff
@@ -127,10 +139,25 @@ export default {
       return UsersRepository.getUser(this.userId)
         .then(response => {
           this.user = response.data;
+          // Keycloak returns attributes as arrays which doesn't work with the autocomplete for org details
+          this.user.attributes.org_details = formatOrganization(this.user.attributes.org_details);
```

Comment:
we were previously doing this in the getter for the vuex state, just moved it here.